lib/open_api_spex/schema.ex

```diff
@@ -402,6 +402,10 @@ defmodule OpenApiSpex.Schema do
   def example(%Schema{type: :string, format: :"date-time"}), do: "2020-04-20T16:20:00Z"
   def example(%Schema{type: :string, format: :uuid}), do: "02ef9c5f-29e6-48fc-9ec3-7ed57ed351f6"
 
+  def example(%Schema{type: :string, pattern: regexp}) when is_binary(regexp), do:
+    Randex.stream(Regex.compile!(regexp))
```

Comment:
If `randex` is an optional dependency, then this should have some runtime check for the `Randex` module being loaded?